packages/dashboard/src/util/browser.ts

```diff
@@ -0,0 +1 @@
+export const isMacLike = /(Mac|iPhone|iPod|iPad)/i.test(navigator.platform);
```

Comment:
[This web API is deprecated](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/platform), and I think there might be a better way to do this - see:
https://stackoverflow.com/questions/3902635/how-does-one-capture-a-macs-command-key-via-javascript
https://stackoverflow.com/questions/24469943/detecting-command-c-keypress

Reply:
This is being used in this scenario which the MDN documentation says is the "least bad option". haha. This is used to display the hotkeys for copy paste. 

> But there is one case where, among the options you could use, navigator.platform may be the least-bad option: When you need to show users advice about whether the modifier key for keyboard shortcuts is the ⌘ command key (found on Apple systems) rather than the ⌃ control key (on non-Apple systems)

Let me add a note above this method though that in other contexts we should have a different detection method like the documentation suggests.